App/screens/photos.tsx

```diff
@@ -197,7 +227,8 @@ const mapStateToProps = (state: RootState): StateProps => {
 
 const mapDispatchToProps = (dispatch: Dispatch<RootAction>): DispatchProps => ({
   queryPhotos: () => dispatch(photosActions.queryCameraRoll.request()),
-  refreshPhotos: () => dispatch(photosActions.refreshPhotos.request(undefined))
+  refreshPhotos: () => dispatch(photosActions.refreshPhotos.request(undefined)),
+  clearProcessingPhotos: () => dispatch(photosActions.clearProcessingPhotos())
```

Comment:
Option from the camera roll photos screen to clear out processing items since any failed ones (which there are plenty of at this point) pile up over time.